pymic/offload_stream.py

```diff
@@ -562,6 +562,9 @@ def invoke(self, kernel, *args):
             raise OffloadError("Cannot invoke kernel, "
                                "library not loaded on device")
 
+        # Map None to nullptr
+        args = [arg if arg is not None else 0 for arg in args]
```

Comment:
This does not pass a true `nullptr` to the kernel, but converts the ```None``` object into a scalar that is then passed to the kernel. I'd prefer to convert this into a 'real' `nullptr` that becomes visible to the kernel.

Here's the idea:

```stream.invoke(lib.some_kernel, None) ```

would then give the following behavior:

```PYMIC_KERNEL
    void some_kernel(const void *pointer) {
        assert(pointer == nullptr);
    }
```

Would this be the behavior you actually want? I'm talking to the developer of the offload engine to see if we could support this.